examples/predict_univariate_pretrained.jl

```diff
@@ -0,0 +1,165 @@
+# Redistributable source code for predicting outputs using
+# pretrained Univariate Random Forest models with the Resonance methodology
+#
+# Authors: Kevin Bonhan and Guilherme Bottino
+# Version: 0.1
+# run script with flag -h to see help and instructions
+# edit constant `tuning_space` to change hyperparameter search grid
+#####
+# 0. Command-line Argument parsing function
+#####
+
+using Pkg; Pkg.activate(replace(ENV["DIRENV_DIR"], '-' => "")); Pkg.instantiate();
+
+using ArgParse
+
+function cli_args(args)
+
+    s = ArgParseSettings("command-line arguments test")
+
+    add_arg_group!(s, "Required arguments", required=true)
+    @add_arg_table! s begin
```

Comment:
We should make this one script with `predict` and `train` commands, and arguments for regression/classification.

Not right now, but eventually (open an issue please)

Reply:
Done (issue opened)